realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -331,6 +331,7 @@ private static void initializeRealm(Realm realm) {
     /**
      * Tries to update a list of existing objects identified by their primary key with new JSON data. If an existing
      * object could not be found in the Realm, a new object will be created. This must happen within a transaction.
+     * If a Realm object property is not present in the JSON object the Realm object property will be {@code null}.
```

Comment:
That is not true for update, it will be ignored. 

What about ?:
```
<p>
If updating a RealmObject and a field is not found in the JSON object, that field will not be updated. If a new RealmObject is created and a field is not found in the JSON object, that field will be assigned the default value for the field type.
```